pkg/bundle/controller.go

```diff
@@ -19,6 +19,7 @@ package bundle
 import (
 	"context"
 	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Nit: This should be moved to the next import block. This block is for stdlib imports.

Reply:
I think my IDE shuffles them automatically. Anyway, its fixed now. Good catch.